static/app/components/charts/baseChart.tsx

```diff
@@ -67,6 +68,8 @@ const handleClick = (clickSeries: any, instance: ECharts) => {
   }
 };
 
+echarts.use(AriaComponent);
```

Comment:
Looks like `echarts` is mocked by `tests/js/sentry-test/echartsMock.js` so this is failing in tests. Will have to add a `use` method it looks like?

Reply:
Ohhhh that's how you do it. Thanks so much! The tests should be passing now – https://github.com/getsentry/sentry/pull/39653/commits/45957c65b195837812e2a0b9455a6952643d5ef9